### ISSUE TITLE
how to get the mAP

### ISSUE THREAD

**Issue by lzg188** [tfzhou/C-HOI#12]
thanks for your jobs,but how can i get the mAP, using the test_hoiw.py only get the hoiw_result.json

**Comment by HawkL327** [comment]
Just add --out and --eval in your command line. Do you have test_2019.json file?

**Comment by lzg188** [comment]
thank u,
not have, i just want to get val dataset result first.

> Just add --out and --eval in your command line. Do you have test_2019.json file?

**Comment by HawkL327** [comment]
Could you share the val dataset result ? My result mIoU 0.5:0.95 is 0.791, that seems unconvinced

**Comment by HawkL327** [comment]
sorry, it's my fault. To evaluate result, you should use the following tool.   https://github.com/YueLiao/PIC_HOIW#use-api

**Comment by tfzhou** [comment]
@HawkL327 @lzg188 ，please see the updated README for test-set annotations and evaluation code. For the val-set result, I am not sure whether the validation set was included for training. Now that you can evaluate on the test set, please use the validation result for reference.
